Expensify/App
# API prep for reconnection callback syncing indicator

### Details
Doing some prep work for E/E 159767

### Fixed Issues (prep work does not fix)
https://github.com/Expensify/Expensify/issues/159767 

### Tests
1. Inspect the Application storage in Chrome Dev Tools and filter for `isLoadingAfterReconnect`
2. Turn off Wi-Fi and wait until indicator turns grey next to avatar
3. Turn on Wi-Fi and verify that `isLoadingAfterReconnect` is set to `true` then eventually changes to `false`

https://user-images.githubusercontent.com/32969087/115919962-3ecfb400-a415-11eb-895a-ed6c5d7b5969.mp4

### QA Steps
1. Test for basic regressions around logging in/out, reports loading, etc

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
